tachyon_core/src/storage/file.rs

```diff
@@ -249,10 +249,7 @@ impl Cursor {
         }
 
         while cursor.current_timestamp < start {
```

Comment:
`current_timestamp` is the timestamp that is returned upon the next call to next(). So we just need to check whether if this is < start to maintain the same logic as before